.github/workflows/test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
       # coursier cache action caches both coursier and sbt caches
       - name: coursier-cache-action
-        uses: coursier/cache-action@v5
+        uses: coursier/cache-action@v6
```

Comment:
There is a `coursier/cache-action@v5` in format.yaml that needs updating: https://github.com/broadinstitute/rawls/blob/af2435c0e833d70a346d65891c7f8c511f3eadb5/.github/workflows/format.yaml#L20